packages/blade/package.json

```diff
@@ -207,7 +208,9 @@
     "style-dictionary": "3.7.1",
     "styled-components": "5.3.5",
     "tsconfig-paths-webpack-plugin": "3.5.2",
-    "jsdom-testing-mocks": "1.9.0"
+    "jsdom-testing-mocks": "1.9.0",
+    "patch-package": "7.0.0",
+    "postinstall-postinstall": "2.1.0"
```

Comment:
what's this sorcery `postinstall-postinstall` 😆

Reply:
They [recommend](https://github.com/ds300/patch-package#why-use-postinstall-postinstall-with-yarn) using this for yarn legacy 😸